src/util-logopenfile.c

```diff
@@ -64,8 +65,10 @@ SCLogOpenUnixSocketFp(const char *path, int sock_type)
     return ret;
 
 err:
-    SCLogError(SC_ERR_SOCKET, "Error connecting to socket \"%s\": %s",
-               path, strerror(errno));
+    if (log_err)
+        SCLogError(SC_ERR_SOCKET,
```

Comment:
It could be, but it seems like an important notification: since an alert (or whatever) may not have been recorded due to the connect failure, error seemed like the right characterization.